src/Domain/Common/CommonEntity.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace CleanArchitecture.Domain.Common
+{
+    public class CommonEntity : AuditableEntity
```

Comment:
I think it would be better to call the class BaseEntity, as it's more of a convention - e.g. https://github.com/dotnet-architecture/eShopOnWeb/blob/main/src/ApplicationCore/Entities/BaseEntity.cs

Reply:
It's not always going to be the case that all entities are auditable entities